modules/cnv-add-disk-to-vm.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template.adoc
+
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+// ifeval::["{context}" == "cnv-edit-vms"]
+// :cnv-vm-add-disk:
+// :object: virtual machine
+// :object-gui: Virtual Machines
+// endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:cnv-vm-add-disk-template:
```

Comment:
This ifdef is declared but not used (I'm pretty sure asciibinder dislikes that). We can remove this line.

Reply:
Thanks, @aburdenthehand! I'll give it a go for both of your suggestions :)